inbm-lib/inbm_common_lib/utility.py

```diff
@@ -233,8 +233,8 @@ def validate_file_type(path: List[str]) -> None:
     """
     logger.debug("Supported file type prefixes: {0}".format(VALID_MAGIC_FILE_TYPE_PREFIXES))
```

Comment:
**Suggestion:** Replace the string concatenation in the logger.debug call with a more efficient f-string. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    logger.debug(f"Supported file type prefixes: {VALID_MAGIC_FILE_TYPE_PREFIXES}")
```